gaphor/diagram/tools/tests/test_shortcut.py

```diff
@@ -20,6 +20,7 @@ def test_delete_selected_items(create, diagram, event_manager):
     assert not diagram.ownedPresentation
 
 
+# TODO: Do we still want this feature?
```

Comment:
@amolenaar @danyeaw @sz332 
This test fails. I did not want to just remove it. I am not sure whether we want to have a feature where we delete diagram's element from the model when a diagram gets deleted.

If you have "Remove unused elements" feature enabled, it performs this cleanup anyways (I tested it after my change); but now we have an option to not delete it when the feature is disabled - which would be expected behavior in my opinion.

I would like to have a discussion on whether I should delete the test because I have removed the feature.